File: week-07/day-2/Workshop 16.5.2023/sharpie.py
class Sharpie:
    color = ''
    width = ''
    inkAmount = 100

    def __init__(self, color: str, width: int):
        self.color = color
        self.width = width
        self.inkAmount = 100

    def use(self):
        self.inkAmount = self.inkAmount-10


sharpie = Sharpie('Blue', 5)

print(sharpie.inkAmount)
sharpie.use()
print(sharpie.inkAmount)

File: week-03/day-4/Workshop 18.4/src/Sharpie/Sharpie.java
package Sharpie;

public class Sharpie {
    private String color;
    private int width;
     private int inkAmount = 100;

    public void use(){
        this.inkAmount = inkAmount -10;
    }

    public Sharpie(String color, int width) {
        this.color = color;
        this.width = width;
    }

    public String getColor() {
        return color;
    }

    public void setColor(String color) {
        this.color = color;
    }

    public int getWidth() {
        return width;
    }

    public void setWidth(int width) {
        this.width = width;
    }

    public int getInkAmount() {
        return inkAmount;
    }

    public void setInkAmount(int inkAmount) {
        this.inkAmount = inkAmount;
    }
}
